### PR TITLE
fix(plain): don't encode binary response

### LINKED ISSUE #450: SVG images are returned base64
requesting a svg returns a base64 encoded string instead of the original text.

caused by https://github.com/adobe/helix-deploy/issues/52

### PR DESCRIPTION
**Pull request by tripodsan**
fixes #450

**Note**: the response from github doesn't need to be base64 encoded if it's binary, since this is now handled by the adapter. in the ideal case, it can just return the response buffer form the github fetch request.

### CODE REVIEW

File: test/static.test.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ describe('Static Delivery Action #integrationtest', () => {
     assert.equal(res.headers['x-static'], 'Raw/Static');
     assert.equal(res.headers['cache-control'], 's-maxage=300, stale-while-revalidate=2592000');
     assert.equal(res.headers['surrogate-key'], 'LiWDcUs5H72QTkGl');
-    assert.equal(res.headers.etag, '"VIXMjwsHOMPLZGtviFum1TGBFMz6zCQKINQseZ6Ub6k="');
+    assert.equal(res.headers.etag, '"zei2DOT55/ukAFvbmB1QwDy2e3KnWuXg41yw2R6TRng="');
```
Comment (tripodsan) [inline test/static.test.js:122]: the hash is different, since we now hash the binary data and not the base64 one.

File: test/static.test.js [hunk 2]
```diff
@@ -136,7 +136,7 @@ describe('Static Delivery Action #integrationtest', () => {
     assert.equal(res.headers['x-static'], 'Raw/Static');
     assert.equal(res.headers['cache-control'], 's-maxage=300, stale-while-revalidate=2592000');
     assert.equal(res.headers['surrogate-key'], 'CIUWTRUuAYPY51zR');
-    assert.equal(res.headers.etag, '"hNCISMWqyUDaDY2zDTMKXb1peyzXunKE791FEtjILtg="');
+    assert.equal(res.headers.etag, '"uc0mBep1KTsWuJKpfF5LC8GPPa/Qy9+JfIAljVdBXIA="');
```
Comment (tripodsan) [inline test/static.test.js:139]: the hash is different, since we now hash the original JSON string and not use object-hash

**Review by trieloff — APPROVED** [review]

**Review by trieloff — COMMENTED** [review]
We should add a post-deploy test for the SVG image, as @davidnuescheler suggested.

**Comment by tripodsan** [comment]
> We should add a post-deploy test for the SVG image, as @davidnuescheler suggested.

I already added tests for svg, ico, and json:
https://github.com/adobe/helix-static/blob/a6cf1cc749bc1dc24b77ad5f97648614ed47aa8b/test/post-deploy.test.js#L63

**Pull request merged**